mllaunchpad/wsgi.py

```diff
@@ -25,24 +25,21 @@
 # necessary to wrap the preparatory code in a try:except: statement.
 try:
     conf = config.get_validated_config()
+except FileNotFoundError:
+    conf = None
 
     # if you change the name of the application variable, you need to
     # specify it explicitly for gunicorn: gunicorn ... launchpad.wsgi:appname
+if conf:
     application = Flask(__name__, root_path=conf["api"].get("root_path"))
-
     ModelApi(conf, application)
-except FileNotFoundError:
-    logger.error(
-        "Config file could not be loaded. Starting the Flask application "
-        "will fail."
-    )
 
-if __name__ == "__main__":
-    logger.warning(
-        "Starting Flask debug server.\nIn production, please use a WSGI server, "
-        + "e.g. 'gunicorn -w 4 -b 127.0.0.1:5000 mllaunchpad.wsgi:application'"
-    )
-    application.run(debug=True)
+    if __name__ == "__main__":
+        logger.warning(
+            "Starting Flask debug server.\nIn production, please use a WSGI server, "
+            + "e.g. 'gunicorn -w 4 -b 127.0.0.1:5000 mllaunchpad.wsgi:application'"
+        )
+        application.run(debug=True)
```

Comment:
Here's a suggestion which you can accept to the Codacy check from failing and complaining that we're running Flask insecurely.
```suggestion
        # Flask apps must not be run in debug mode in production, because this allows for arbitrary code execution.
        # We know that and advise the user that this is only for debugging, so this is not a security issue (marked nosec):
        application.run(debug=True)  # nosec
```